PayPal/src/main/java/com/braintreepayments/api/paypal/PayPalClient.kt

```diff
@@ -54,8 +56,9 @@ class PayPalClient internal constructor(
     constructor(
         context: Context,
         authorization: String,
-        appLinkReturnUrl: Uri
-    ) : this(BraintreeClient(context, authorization, null, appLinkReturnUrl))
+        appLinkReturnUrl: Uri,
+        returnUrlScheme: String? = null
```

Comment:
Good call, let me update the kdoc.

Reply:
Do we need to consider any type of okay from security for adding this back? I know we were moving towards removing url schemes across the SDKs for security reasons but wasn't sure what applied with it as an opt in.

Reply:
I don't think we need to run this by security as our other modules are still relying on deep links for navigating back to the merchant app. I also don't think security originally required the switch to app links. @sarahkoop - do you have context on if security was originally involved with the switch to app links?

Reply:
Yes I think since we are still supporting deep links across multiple flows it's fine to add back in - this originated as a merchant request not an internal security mandate